Sources/Sentry/SentryBreadcrumbTracker.m

```diff
@@ -141,13 +141,32 @@ - (void)addEnabledCrumb
     [SentrySDK addBreadcrumb:crumb];
 }
 
-- (void)swizzleSendAction
++ (BOOL)avoidSender:(id)sender forTarget:(id)target action:(NSString *)action
 {
 #if SENTRY_HAS_UIKIT
+    if ([sender isKindOfClass:UITextField.self]) {
+        // This is required to avoid creating breadcrumbs for every key pressed in a text field.
+        // Textfield may invoke many types of event, in order to check if is a
+        // `UIControlEventEditingChanged` we need to compare the current action to all events
+        // attached to the control. This may cause a false negative if the developer is using the
+        // same action for different events.
```

Comment:
> This may cause a false negative if the developer is using the same action for different events.

`h`: We know the type of action in `swizzleSendAction`. Can't we pass it to this function and check if this action == UIControlEventEditingChanged?

Reply:
Action is the name of the function being called. Event type can be found in the `event` argument, but for this event, the argument is nil.

Reply:
> This may cause a false negative if the developer is using the same action for different events.

I'm worried about this ⬆️ . Can't we do anything to avoid this? Maybe we can discuss this on a call.